### PR TITLE
You now only shout if you use !! instead of ! + apostrophe autopunctuation.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Doing `Hello!` will now always make you exclaim.
Doing `Help!!` will now always make you scream/shout/yell.
Same with doing `Help?!`, `Help!?`.

Also, doing `Bruh'` now autopunctuates.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/mob/language/generic.dm [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	name = LANGUAGE_TCB
 	desc = "A spiritual successor of Esperanto, established in 2404 in Tau Ceti by Ceti intellectuals. Its unique, fully customized alphabet and structure allow it to be spoken even by most alien species. It's the official language of Tau Ceti and has growing traction in diplomatic circles and Universalists across human space."
 	speech_verb = "says"
```
Comment (mikomyazaki) [inline code/modules/mob/language/generic.dm:25]: Why doesn't speech_verb have to be a list, but all the other ones do?

File: code/modules/mob/language/language.dm [hunk 2]
```diff
@@ -8,9 +8,10 @@
...
 	var/desc = "A language."          // Short description for 'Check Languages'.
 	var/speech_verb = "says"          // 'says', 'hisses', 'farts'.
-	var/ask_verb = "asks"             // Used when sentence ends in a ?
-	var/exclaim_verb = "exclaims"     // Used when sentence ends in a !
-	var/whisper_verb                  // Optional. When not specified speech_verb + quietly/softly is used instead.
+	var/list/ask_verb = list("asks")  // Used when sentence ends in a ?
+	var/list/exclaim_verb = list("exclaims") // Used when sentence ends in a !
+	var/list/shout_verb = list("shouts", "yells", "screams") //Used when a sentence ends in !!
+	var/list/whisper_verb = list("says quietly", "says softly", "whispers")  // Optional. When not specified speech_verb + quietly/softly is used instead.
 	var/signlang_verb = list("signs") // list of emotes that might be displayed if this language has NONVERBAL or SIGNLANG flags
```
Comment (mikomyazaki) [inline code/modules/mob/language/language.dm:15]:
```suggestion
	var/list/signlang_verb = list("signs") // list of emotes that might be displayed if this language has NONVERBAL or SIGNLANG flags
```

File: code/modules/mob/say.dm [hunk 3]
```diff
@@ -111,14 +111,16 @@
...
-        else if(ending=="?")
-                verb="asks"
-
-        return verb
+	. = "says"
+	var/ending = copytext(message, length(message))
+	var/pre_ending = copytext(message, length(message) - 1, length(message))
+	if(ending == "!")
+		if(pre_ending == "!" || pre_ending == "?")
+			. = pick("shouts", "yells")
```
Comment (mikomyazaki) [inline code/modules/mob/say.dm:119]: Shouldn't these verbs be based off the language verbs? And same on 121. Why are they hard-coded here?

Comment (mikomyazaki) [reply code/modules/mob/say.dm:119]: Infact, shouldn't this proc just call /datum/language/proc/get_spoken_verb(var/msg_end, var/pre_end)?

File: code/modules/mob/language/generic.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/mob/language/generic.dm]: I was fixing this actually lol

File: code/modules/mob/language/language.dm [hunk 4]
```diff
@@ -7,12 +7,12 @@
 /datum/language
 	var/name = "an unknown language"  // Fluff name of language if any.
 	var/desc = "A language."          // Short description for 'Check Languages'.
-	var/speech_verb = "says"          // 'says', 'hisses', 'farts'.
+	var/speech_verb = list("says")          // 'says', 'hisses', 'farts'.)
```
Comment (mikomyazaki) [inline code/modules/mob/language/language.dm:10]:
```suggestion
	var/list/speech_verb = list("says")          // 'says', 'hisses', 'farts'.)
```

**Pull request merged**